package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "prebuild": "npm install @laserfiche/types-lf-ui-components@12",
+    "install-lf-documentation": "npm install ./projects/lf-documentation --no-save",
```

Comment:
what does --no-save mean?

Reply:
no-save does not save it to root package.json. without no save, it will install a node_module of this local npm package in the node_modules as well as a link to that in package.json of the root